paystack/paystack.php

```diff
@@ -314,12 +318,12 @@ public function checkCurrency($cart)
         }
         return false;
     }
-
-    public function checkCurrencyNGN($cart)
+	
+	public function checkCurrencyNGN($cart)
     {
         $currency_order = new Currency($cart->id_currency);
-        if ($currency_order->iso_code == 'NGN' || $currency_order->iso_code == 'GHS' || $currency_order->iso_code == 'ZAR' || $currency_order->iso_code == 'USD' || $currency_order->iso_code == 'XOF' || $currency_order->iso_code == 'KES' || $currency_order->iso_code == 'EGP') {
-            return true;
+        if ($currency_order->iso_code == 'NGN') {
```

Comment:
@RooseveltTech it looks like you've removed a bunch of the other currencies the plugin supports here

Reply:
I can also add that back

Reply:
Yeah. You can leave out EGP though

Reply:
alright